lib/modules/manager/hermit/artifacts.spec.ts

```diff
@@ -0,0 +1,295 @@
+import type { StatusResult } from 'simple-git/promise';
+import { mockedFunction, partial } from '../../../../test/util';
+import { GlobalConfig } from '../../../config/global';
+import { exec } from '../../../util/exec';
+import { localPathIsSymbolicLink, readLocalSymlink } from '../../../util/fs';
+import { getRepoStatus } from '../../../util/git';
+import type { UpdateArtifact } from '../types';
+import { updateArtifacts } from '.';
+
+jest.mock('../../../util/exec');
+jest.mock('../../../util/git');
+jest.mock('../../../util/fs');
+
+const execMock = mockedFunction(exec);
+const getRepoStatusMock = mockedFunction(getRepoStatus);
+
+const lstatsMock = mockedFunction(localPathIsSymbolicLink);
+const readlinkMock = mockedFunction(readLocalSymlink);
+
+describe('modules/manager/hermit/artifacts', () => {
+  describe('updateArtifacts', () => {
+    it('should run hermit install for packages and return updated files', async () => {
+      lstatsMock.mockResolvedValue(true);
+
+      readlinkMock.mockResolvedValue('hermit');
+      GlobalConfig.set({ localDir: '' });
+
+      execMock.mockResolvedValue({
```

Comment:
```suggestion
      execMock.mockResolvedValueOnce({
```

So it doesn't interfere with other tests or reset mocks in `beforeEach` function